.github/workflows/ci.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v1
       - name: Setup Minikube
-        uses: manusa/actions-setup-minikube@v2.3.1
+        uses: manusa/actions-setup-minikube@v2.4.2
```

Comment:
action version update brought something specific to updating minikube version?

Reply:
Nothing specific, just updated to the latest release version